csrankings-c.csv

```diff
@@ -1039,6 +1039,7 @@ Chu-Xu Zhang,Brandeis University,https://chuxuzhang.github.io,ZbZ0l0oAAAAJ
 Chuadhry Mujeeb Ahmed,University of Strathclyde,https://pureportal.strath.ac.uk/en/persons/chuadhry-mujeeb-ahmed/publications,l_qJD3IAAAAJ
 Chuan Chen 0001,Sun Yat-sen University,http://106.52.19.28/faculty/index.html,yJId-nEAAAAJ
 Chuan Lu,Aberystwyth University,https://www.aber.ac.uk/en/cs/staff-profiles/listing/profile/cul,tvY6seQAAAAJ
+Chuan Luo,Beihang University,https://soft.buaa.edu.cn/teachershouw.jsp?urltype=news.NewsContentUrl&wbtreeid=1262&wbnewsid=9648,MfJGmUUAAAAJ

```

Comment:
```
Processing Chuan Luo,Beihang University,https://soft.buaa.edu.cn/teachershouw.jsp?urltype=news.NewsContentUrl&wbtreeid=1262&wbnewsid=9648,MfJGmUUAAAAJ
  Invalid name (Chuan Luo). This may be a disambiguation entry.
```

Reply:
Indeed, Chuan Luo may point to different persons but there is only one entry on DBLP. Any suggestion on how should we do it in this case?

Reply:
This is a disambiguation page. Please read the instructions for what to do with DBLP.org; in the meanwhile, this entry must be removed.